docker/docker_image_src.go

```diff
@@ -187,6 +187,7 @@ func (s *dockerImageSource) fetchManifest(ctx context.Context, tagOrDigest strin
 		"Accept": manifest.DefaultRequestedManifestMIMETypes,
 	}
 	res, err := s.c.makeRequest(ctx, "GET", path, headers, nil, v2Auth, nil)
+	logrus.Debugf("Content-Type from GET to %s is %s", path, res.Header.Get("Content-Type"))
```

Comment:
tests aren't happy.  I'm guessing you might want to add a nil check on ref before derefencing?

Reply:
Yes this message should only be set on success.

Reply:
Whoops, you're right, let me fix that.

Reply:
Fixed to set the log only after we check success.

Reply:
Checks fixed, terribly sorry about that, rookie mistake.